Modal.js

```diff
@@ -0,0 +1,172 @@
+export class Modal {
+
+    modal = `
+        <div class="modal is-active">
+            <div class="modal-background"></div>
+            <div class="modal-card">
+                <header class="modal-card-head py-3">
```

Comment:
Your layout here uses [Bulma spacing helpers](https://bulma.io/documentation/helpers/spacing-helpers/). I'm glad that you found them, but I don't want you to use them. Bulma is a framework for a reason: someone way better than either of us at graphic design looked at all the possible situations (including mobile/desktop/etc layouts) and created default, consistent spacing across the Bulma elements. While we can adjust if we have a _really_ good reason to do so, I think it makes the most sense to stick to defaults.

I didn't like this when I saw it yesterday, but I couldn't quite explain why. Funnily enough, [this typography video](https://www.youtube.com/watch?v=9ElrcTtAxzA&list=LL) came up on my recommended this morning and I decided to watch it. In this video, the narrator shows why this isn't a good idea when it comes to visual weight.

If you would, please remove Bulma spacing helpers across the code unless you have a really good reason to use them.

Reply:
Done! It hurt my eyes too!